### PR TITLE
Fix entity active states

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

There was a typo for the `media_player` domain and the `script` rules was wrong.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/common/entity/state_active.ts [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
   const domain = computeDomain(stateObj.entity_id);
   const state = stateObj.state;
 
-  if (
-    OFF_STATES.includes(state) &&
-    !(NORMAL_UNKNOWN_DOMAIN.includes(domain) && state === "unknown") &&
-    !(NORMAL_OFF_DOMAIN.includes(domain) && state === "script")
-  ) {
+  if (["button", "input_button", "scene"].includes(domain)) {
+    return state !== UNAVAILABLE;
```
Comment (bramkragten) [inline src/common/entity/state_active.ts:10]: So these are always active?

**Review by bramkragten — DISMISSED** [review]

File: src/common/entity/state_active.ts (hunk 1 above)
Comment (piitaya) [reply src/common/entity/state_active.ts:10]: Yes, just inactive if there are unavailable.

File: src/common/entity/state_active.ts [hunk 2]
```diff
@@ -24,7 +21,7 @@ export function stateActive(stateObj: HassEntity, state?: string): boolean {
     case "device_tracker":
     case "person":
       return compareState !== "not_home";
-    case "media-player":
+    case "media_player":
       return compareState !== "idle" && compareState !== "standby";
```
Comment (piitaya) [inline src/common/entity/state_active.ts:25]: @bramkragten Do you think we must consider `idle` state as active ? According to core documentation, the device is turned on but no media or unknown media (e.g. HDMI input for receiver ?) https://developers.home-assistant.io/docs/core/entity/media-player/#states)

Comment (bramkragten) [reply src/common/entity/state_active.ts:25]: I think it depends on the device, for a receiver, probably yes? For a chromecast, probably no?

Comment (piitaya) [reply src/common/entity/state_active.ts:25]: May be we can use the device class :
idle active for `receiver`
idle inactive for `tv` and `speaker`

Comment (piitaya) [reply src/common/entity/state_active.ts:25]: After multiple discussion with users, it's better to consider it "active" as the device is turned on

**Review by bramkragten — APPROVED** [review]

**Pull request merged**